addons/im_livechat/tests/chatbot_common.py

```diff
@@ -140,6 +140,6 @@ def _post_answer_and_trigger_next_step(cls, mail_channel, answer, chatbot_script
             cls.env['chatbot.message'].search([
                 ('mail_message_id', '=', mail_message.id)
             ], limit=1).user_script_answer_id = chatbot_script_answer.id
-
-        next_step = mail_channel.chatbot_current_step_id._process_answer(mail_channel, mail_message.body)
+        # sudo: The response step is run by system and should be in sudo mode to skip ACLs
```

Comment:
This is not explained in the PR message. What flow is it fixing? Same remark about sudo comment.

Reply:
This is just a fix for the unit test. In real, the `_process_answer` is always called in sudo mode, but in the test it's not. So I add sudo here to simulate the real cases.